test/integration/rules/autocomplete-valid/autocomplete-valid.json

```diff
@@ -71,6 +71,9 @@
     ["#pass60"],
     ["#pass61"],
     ["#pass62"],
-    ["#pass63"]
+		["#pass63"],
+		["#pass64"],
+		["#pass65"],
+		["#pass66"]
```

Comment:
FYI, formatting looks spotless in IDE. Github seems to complain on indentation here.

![image](https://user-images.githubusercontent.com/20978252/45148806-06c1c700-b1c0-11e8-8814-49639b35b4d0.png)